jwst/engdblog/tests/test_engdblog.py

```diff
@@ -32,7 +32,7 @@ def test_barestring(caplog, engdb):
     assert mnemonic in result
     assert 'EngDBLogStep instance created' in caplog.text
     assert mnemonic in caplog.text
-    assert "Step EngDBLogStep running with args ('INRSI_GWA_Y_TILT_AVGED',).".format(mnemonic) in caplog.text
+    assert f"Step EngDBLogStep running with args ('{mnemonic}')." in caplog.text
```

Comment:
Was the original line including the mnemonic name twice: once in the hardcoded 'INRSI_GWA_Y_TILT_AVGED' and then a second time via '.format(mnemonic)'?

Reply:
Good question.  I think it should be included once.  Hard to know, as this test is set to `xfail`.

Reply:
just a buggy line never checked for due to the specified `xfail`. Fix is correct.